backends/qualcomm/tests/models.py

```diff
@@ -440,6 +451,30 @@ def forward(self, x):
         return x / 10
 
 
+class EinsumBilinear(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, bn, anm, bm):
+        return torch.einsum("bn,anm,bm->ba", bn, anm, bm)
+
+
+class EinsumOuterProduct(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, i, j):
+        return torch.einsum("i,j->ij", i, j)
```

Comment:
Is einsum used somewhere?

Reply:
We saw `einsum` in  MOEFeedForward under `executorch.examples.models.llama2.llama_transformer`

Reply:
Ah I see, so actually the `einsum` is probably not the best for qnn backend, it's better to replace with other way (I try it before) to get a better performance. But it's still good to run `einsum`.